scripts/firedrake-install

```diff
@@ -524,7 +538,7 @@ elif not (args.system or args.user or args.prefix):
         quit("Virtual env installed. Please run firedrake-install again.\n")
     if mode == "install":
         stdout.write("Creating firedrake virtualenv.\n")
-        virtualenv.create_environment("firedrake", site_packages=True)
+        virtualenv.create_environment("firedrake", site_packages=False)
```

Comment:
Yes. We're hoping this will make the install script more robust (at pretty minimal cost).